server/controllers/refer/referralsController.test.ts

```diff
@@ -36,8 +33,15 @@ describe('ReferralsController', () => {
   const courseOffering = courseOfferingFactory.build()
 
   let referralsController: ReferralsController
+  let request: DeepMocked<Request>
+  let response: DeepMocked<Response>
+  let next: DeepMocked<NextFunction>
```

Comment:
May not be necessary for `next`

Reply:
Also open to alternative methods, but this is how I've done it previously.  Set it as a let and then reset it in the `beforeEach`